src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -122,68 +113,131 @@ public static void tearDown() {
         apiUser = Persistence.apiUsers.getById(apiUser.id);
         if (apiUser != null) apiUser.delete();
         otpUser = Persistence.otpUsers.getById(otpUser.id);
-        if (otpUser != null) otpUser.delete();
+        if (otpUser != null) otpUser.delete(false);
     }
 
     /**
      * Tests to confirm that an otp user, related monitored trip and plan can be created and deleted leaving no orphaned
      * records. This also includes Auth0 users if auth is enabled.
      */
     @Test
-    public void canSimulateApiUserFlow() throws IOException, URISyntaxException {
-        // create otp user as api user
+    public void canSimulateApiUserFlow() throws URISyntaxException {
+
+        // obtain Auth0 token for Api user.
+        String endpoint = String.format("api/secure/application/%s/authenticate?username=%s&password=%s",
+            apiUser.id,
+            apiUser.email,
+            TEMP_AUTH0_USER_PASSWORD);
+        HttpResponse<String> getTokenResponse = mockAuthenticatedPost(endpoint,
```

Comment:
This call should use a mock call that does not provide an Authorization header to the authentication endpoint.

Reply:
Updated.